oneanddone/base/static/css/one-and-done.less

```diff
@@ -166,7 +166,7 @@ footer {
 }
 .feedback-form {
     width: 95%;
-    text-align: right;
+    text-align: left;
```

Comment:
While this makes the new text input appear properly aligned, it also causes the "No Thanks" and "Submit Feedback" buttons to now appear in the lower left of the form, as opposed to the lower right, which is where they are on the current site, and where they need to stay, so this is not correct.

I would suggest that any time you are changing existing css (such as here, where you are changing an existing property from `right` to `left`), you need to be very careful and review the impact of the change on the site. This is something you could have noticed if you would have compared the feedback form currently on the site and the new feedback form produced with this change.